src/components/app/data/hooks/useSubscriptions.js

```diff
@@ -16,10 +15,7 @@ export default function useSubscriptions(queryOptions = {}) {
     bffQueryOptions: {
       ...queryOptionsRest,
       select: (data) => {
-        const transformedData = transformSubscriptionsData(
-          data?.enterpriseCustomerUserSubsidies?.subscriptions,
-          { isBFFData: true },
-        );
+        const transformedData = data?.enterpriseCustomerUserSubsidies?.subscriptions;
```

Comment:
[inform] The BFF's `subscriptions` data will now already have the expected transforms. No need to pass the subscriptions data to `transformSubscriptionsData` anymore.